src/test/java/com/google/cloud/flink/bigquery/ProtobufUtilsTest.java

```diff
@@ -0,0 +1,334 @@
+    assertThat(protoSchema).isNotNull();
+    assertThat(descriptor).isNotNull();
+    assertThat(descriptor.getFields().get(0).getName()).isEqualTo("id");
+    assertThat(descriptor.getFields().get(1).getName()).isEqualTo("name");
+  }
+
+  @Test
+  public void buildSingleRowMessageRowTypeTest() throws IOException, DescriptorValidationException {
+    List<RowField> fields = new ArrayList<RowField>();
+    fields.add(new RowField("id", DataTypes.BIGINT().getLogicalType()));
+    fields.add(new RowField("name", DataTypes.STRING().getLogicalType()));
+    RowType rowType = new RowType(fields);
+    Row row = new Row(3);
+    row.setKind(RowKind.INSERT);
+    row.setField(0, 100);
+    row.setField(1, "John");
+    Descriptor descriptor = toDescriptor(rowType);
+    DynamicMessage dynamicMessage = ProtobufUtils.buildSingleRowMessage(rowType, descriptor, row);
+    Set<FieldDescriptor> keySet = dynamicMessage.getAllFields().keySet();
+    Object[] keySetArray = keySet.toArray();
+    assertThat(rowType).isNotNull();
+    assertThat(descriptor).isNotNull();
+    assertThat(dynamicMessage).isNotNull();
+    assertThat(keySet).isNotNull();
+    assertThat(dynamicMessage.getAllFields().size()).isEqualTo(2);
+    assertThat(((Long) dynamicMessage.getAllFields().get(keySetArray[0])).intValue())
+        .isEqualTo(100);
+    assertThat(dynamicMessage.getAllFields().get(keySetArray[1])).isEqualTo("John");
+  }
+
+  @Test
+  public void buildSingleRowMessageStructTypeTest()
+      throws IOException, DescriptorValidationException {
+    StructuredType.Builder builder = StructuredType.newBuilder(Row.class);
+    ArrayList<StructuredAttribute> listOfAttribute = new ArrayList<StructuredAttribute>();
+    listOfAttribute.add(new StructuredAttribute("id", DataTypes.BIGINT().getLogicalType()));
+    listOfAttribute.add(new StructuredAttribute("name", DataTypes.STRING().getLogicalType()));
+    builder.attributes(listOfAttribute);
+    builder.setFinal(true);
+    builder.setInstantiable(true);
+    StructuredType structType = builder.build();
+    Row row = new Row(3);
+    row.setKind(RowKind.INSERT);
+    row.setField(0, 100);
+    row.setField(1, "John");
+    Descriptor descriptor = toDescriptor(structType);
+    DynamicMessage dynamicMessage =
+        ProtobufUtils.buildSingleRowMessage(structType, descriptor, row);
+    Set<FieldDescriptor> keySet = dynamicMessage.getAllFields().keySet();
+    Object[] keySetArray = keySet.toArray();
+
+    assertThat(structType).isNotNull();
+    assertThat(descriptor).isNotNull();
+    assertThat(dynamicMessage).isNotNull();
+    assertThat(keySet).isNotNull();
+
+    assertThat(dynamicMessage.getAllFields().size()).isEqualTo(2);
+    assertThat(((Long) dynamicMessage.getAllFields().get(keySetArray[0])).intValue())
+        .isEqualTo(100);
+    assertThat(dynamicMessage.getAllFields().get(keySetArray[1])).isEqualTo("John");
+  }
+
+  @Test
+  public void LogicalTypeToProtoTypeConverterTest() {
```

Comment:
Why do we have this test in the first place? We are not testing any functionality in the ProtobufUtils class here